src/slack.ts

```diff
@@ -37,8 +37,7 @@ export const COLOURS = {
     neutral: '#A8A8A8'
 };
 
-export async function postMessage(message: SlackMessage): Promise<number> {
-    const slackWebhookUrl = process.env.SLACK_WEBHOOK;
+export async function postMessage(message: SlackMessage, slackWebhookUrl: string | undefined): Promise<number> {
```

Comment:
Instead of passing the webhook URL every time maybe we want a function with default of the INCIDENTS channel and only provide it if you want to send to CICD channel.

95% of t he time we will want to send to the incidents channel.